packages/controller-utils/src/util.test.ts

```diff
@@ -11,6 +12,14 @@ describe('util', () => {
     nock.cleanAll();
   });
 
+  it('isSafeChainId', () => {
+    expect(util.isSafeChainId(MAX_SAFE_CHAIN_ID + 1)).toBe(false);
+    expect(util.isSafeChainId(MAX_SAFE_CHAIN_ID)).toBe(true);
+    expect(util.isSafeChainId(-1)).toBe(false);
```

Comment:
Nit: Maybe zero would be a better test case here? Since zero is the boundary, not -1

```suggestion
    expect(util.isSafeChainId(0)).toBe(false);
```